corehq/apps/app_manager/templates/app_manager/partials/case_list_lookup.html

```diff
@@ -122,9 +123,7 @@
                         {% trans "Results display text" %}
                     </label>
                     <div class="col-sm-4">
-                        <input class="form-control" type="text"
-                               data-bind="value: lookup_field_header"
-                               placeholder="{% trans 'Accuracy' %}" />
+                        {% input_trans detail.short.lookup_field_header langs 'lookup_field_header' data_bind='value: lookup_field_header' %}
```

Comment:
hrm, I thought it should just work. might be because of the knockout...there's also a js version of the "input + lang button" widget (used in case list/detail property list config) which might work better here, though i don't know offhand how reusable it is (might help that you're already on the case list config). i can take a closer look in a day or two.